tests/helpers.py

```diff
@@ -589,7 +589,7 @@ def create_pvc(
 
 
 def create_multiple_pvcs(
-    sc_name, namespace, number_of_pvc=1, size=None, do_reload=False,
+    sc_name, namespace, number_of_pvc=1, size=None,
```

Comment:
if we can add bulk option then we can keep original signature for other cases.

Reply:
done